tests/testthat/test-epi_slide.R

```diff
@@ -677,3 +955,385 @@ test_that("epi_slide gets correct ref_time_value when groups have non-overlappin
 
   expect_identical(result1, expected_output)
 })
+
+test_that("results for different `before`s and `after`s match between epi_slide and epi_slide_mean", {
+  # 3 missing dates
+  n <- 15 # Max date index
+  m <- 3 # Number of missing dates
+  n_obs <- n + 1 - m # Number of obs created
+  k <- c(0:(n - (m + 1)), n) # Date indices
+
+  # Basic time type
+  days <- as.Date("2022-01-01") + k
```

Comment:
I'd maybe put this nearer the tests its used with, as this location suggests it's getting used in the definition of `tes_time_type_mean` but nbd really.